lib/elixir/src/elixir_parser.yrl

```diff
@@ -857,6 +857,21 @@ warn_empty_stab_clause({stab_op, {Line, _Begin, _End}, '->'}) ->
     "an expression is always required on the right side of ->. "
     "Please provide a value after ->").
 
+%% TODO: Make this an error on Elixir v2.0.
+warn_unary_operator_eol({dual_op, {Line, _Begin, _End}, Op}) ->
+  elixir_errors:warn(Line, ?file(),
+    io_lib:format(
+      "unary operator ~ts followed by new line. "
+      "This sometimes happen when you try to use a binary operator in multiple lines, "
```

Comment:
I think it would be more correct if it said: "This sometimes happen when you try to use a binary operator over multiple lines. "

But I am not a native speaker, thoughts?